projects/core/directives/dropdown/dropdown-context.directive.ts

```diff
@@ -28,24 +25,17 @@ const MOVE_THRESHOLD = 15;
         '[style.user-select]': 'userSelect()',
         '[style.-webkit-user-select]': 'userSelect()',
         '[style.-webkit-touch-callout]': 'userSelect()',
-        '(touchend.silent.passive)': 'onTouchEnd()',
-        '(touchcancel.silent.passive)': 'onTouchEnd()',
-        '(touchmove.silent.passive)':
-            'onTouchMove($event.touches[0].clientX, $event.touches[0].clientY)',
-        '(touchstart.silent.passive)':
-            'onTouchStart($event.touches[0].clientX, $event.touches[0].clientY)',
         '(document:pointerdown.silent)': 'closeDropdown($event)',
         '(document:contextmenu.capture.silent)': 'closeDropdown($event)',
         '(document:keydown.esc)': 'closeDropdown()',
-        '(contextmenu.prevent.stop)': 'onContextMenu($event.clientX, $event.clientY)',
+        // cspell:disable-next-line
```

Comment:
let's add `longtap` to ignore list and remove comment 
https://github.com/taiga-family/taiga-ui/blob/main/.cspell.json

Reply:
<img width="538" alt="image" src="https://github.com/user-attachments/assets/d65366fb-82c0-45fa-94c5-c8ebd800d28c">

https://github.com/taiga-family/configurations/pull/650

Reply:
just remove cspell-disable-next-line comment then